### PR TITLE
Add login

### PR DESCRIPTION
**Pull request by dtbielejeski**
Added login functionality
Calls to the database will need to be updated

### CODE REVIEW

File: ClassySchedule/src/main/java/database/DatabaseStatic.java [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+    /**
+     * Verifies the given credentials. Populates the token object to be used
+     * for future database requests
+     * @param username the username to be checked
+     * @param password the password to be checked
+     * @return true if the user is valid, false otherwise
+     */
+    public static boolean login(String username, String password) {
+        try {
+        CloseableHttpClient client = HttpClients.createDefault();
```
Comment (xion6136) [inline ClassySchedule/src/main/java/database/DatabaseStatic.java]: Should there be index? from within the scope?

File: ClassySchedule/src/main/java/database/DatabaseStatic.java [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+        URIBuilder builder = new URIBuilder(url + "login");
+        JSONObject json = new JSONObject();
+        json.put("password", password);
+        json.put("username", username);
+        StringEntity entity = new StringEntity(json.toString());
+        HttpPost httpPost = new HttpPost(builder.build());
+        httpPost.setHeader("Content-Type", "application/json");
+        httpPost.setEntity(entity);
+        CloseableHttpResponse response = client.execute(httpPost);
+        if(response.getCode() == 200) {
```
Comment (xion6136) [inline ClassySchedule/src/main/java/database/DatabaseStatic.java]: space if ()

**Review by xion6136 — CHANGES_REQUESTED** [review]
I think just some syntax guidelines that needed some changes.

File: ClassySchedule/src/main/java/database/DatabaseStatic.java (hunk 1 above)
Comment (dtbielejeski) [reply ClassySchedule/src/main/java/database/DatabaseStatic.java]: What do you mean?

**Review by xion6136 — APPROVED** [review]

**Review by ZackSchuster600 — APPROVED** [review]
nice code

**Pull request merged**